_posts/0000-01-02-makerverde2020.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
*lets look at the mark down syntax-one astrix*
**two astrix make it bold**
*using astrix for a list
*second list item
*thrid list item